gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -141,7 +141,18 @@ def get_joint_freq_and_faf(
     :return: Table with joint genomes and exomes frequency and FAF information.
     """
     logger.info("Performing an inner join on frequency HTs...")
-    ht = genomes_ht.join(exomes_ht, how="outer")
+    ht = genomes_ht.join(exomes_ht, how="inner")
```

Comment:
I think the logger is wrong, not the code.

Reply:
Why don't we get freq for variants in both exomes and genomes?

Reply:
An outer join will grab any row in either exomes or gneomes. An inner join only takes rows that appear in both.

Reply:
Btw, could you check my slack message? I want to find why we're getting negative values again in `merge_freq_arrays()`, Thanks!

Reply:
No worries at all -- I think it was never officially settled on so the logger being in conflict with the code is very confusing. I shouldve caught this in my original review. I'm discussing this with KC now

Reply:
thank you both for flagging this -- Mike is right in that this wasn't completely settled. from old conversations with Grace, it sounds like we want to have an outer join here for maximum utility. 

we want to have a joint FAF number displayed for variants present in both the exomes and genomes so that users can use a single value (rather than having to decide whether to use the exomes FAF or the genomes FAF, or figure out how to combine these on their own). variants that are only present in one data type do not need a joint FAF, but for ease of use, it would make sense to present them within the same field, so that users don't have to go through this process: is joint FAF defined? -> which one of exomes or genomes FAF is defined?

also adding a note that, we should add a `data_type` field to our joint FAF structs
![image](https://github.com/broadinstitute/gnomad_qc/assets/19142091/bd0146ca-f915-4afd-828a-5d1bbeee9d55)

Reply:
We might as well since its only a logger change right now?

Reply:
Okay, I will send it in.

Reply:
Ok, I can also work on this if you don't have the bandwidth right now

Reply:
@ch-kr -- joint FAF is an array or structs. Do we want dataype on every entry in FAF?  Or are we really after joint_fafmax here?

Reply:
I made the change but I may have underestimate the complicacity?

Reply:
Sorry see -- https://github.com/broadinstitute/gnomad_qc/pull/490/files#r1362611232

Reply:
> @ch-kr -- joint FAF is an array or structs. Do we want dataype on every entry in FAF? Or are we really after joint_fafmax here?

good point. I think just joint_fafmax needs the data type -- this is the field I expect users would want to use

Reply:
sorry, just seeing this now. The inner join in the logger is because I wrote this script originally just for analysis purposes and we filtered to only variants that PASS in both datasets. I modified to be what we actually want, but in the rush I didn't update the logger